test/realm_test.dart

```diff
@@ -1109,6 +1278,50 @@ void openEncryptedRealm(List<int>? encryptionKey, List<int>? decryptionKey, {voi
   }
 }
 
+extension _FutureRealm on Future<Realm> {
```

Comment:
```suggestion
extension on Future<Realm> {
```
this will make the extension private